server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
@chriscartlidge nope :( early December is the next time I can guarantee if you're around?